lib/default.nix

```diff
@@ -188,6 +188,16 @@ in {
   # Useful for pre-filtered package-set.
   collectChecks' = collectChecks (_: true);
 
+  # Flatten the result of collectChecks or collectChecks' for use in flake `checks`
+  flattenChecks = allChecks: builtins.listToAttrs (
```

Comment:
I guess for consistency I'd either call this `mkFlakeChecks`, or also define `flattenTests` etc. and use them in the definitions below, :shrug:

Reply:
Renamed